README.md

```diff
@@ -10,7 +10,7 @@ In today's data-driven world, where data analytics is used by the Researchers or
 This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, helping us achieve better privacy.
 
 **Things to remember about PyDP :**
-- :rocket: Features Differentially Private Algorithmic functions as part of the 0.1.1 dev release, added to carrots demo such as BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median , etc.  
+- :rocket: Features Differentially Private Algorithmic functions to support carrots demo including: BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median, etc.  
 - All the computation methods mentioned above, use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
```

Comment:
Let's add a link for the others mechanism or paper so people have an idea about what does it means:
- https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf
- https://en.wikipedia.org/wiki/Additive_noise_mechanisms

Reply:
Great, I think we can add it to the learning resources @jeamick  - to know more about DP and stuff. The readme should only describe the project. What do you think @chinmayshah99  ?

I think the current description is enough. But, as always any suggestions are welcome!

Reply:
Yeah @shaistha24 , we can add this to learning resources, and create a link from here in readme. 
@shaistha24 Can you help us out here?